src/main/java/org/jpeek/graph/XmlMethodCall.java

```diff
@@ -55,7 +52,7 @@
     @Override
     public String asString() throws IOException {
         return new Joined(
-            "", this.call.xpath("name/text()").get(0),
+            "", this.call.xpath("op/name/text()").get(0),
```

Comment:
@vzurauskas Could you please replace `get(0)` with
`org.cactoos.scalar.ItemAt `?

Reply:
@fanifieiev Can you explain why? If the goal is to get more object oriented by replacing method calls with object composition, this will not help, because `ItemAt` is a `Scalar`, and we would still need to call `value()` on it. This would only introduce an indirection without reducing the number of method calls.

Reply:
@vzurauskas Agree